Library/Formula/rem.rb

```diff
@@ -0,0 +1,15 @@
+class Rem < Formula
+  desc "Command-line tool to access OSX Reminders.app database"
+  homepage "https://github.com/kykim/rem"
+  url "https://github.com/kykim/rem/archive/20150618.tar.gz"
+  sha256 "e57173a26d2071692d72f3374e36444ad0b294c1284e3b28706ff3dbe38ce8af"
+
+  def install
+    xcodebuild
```

Comment:
Yep, [origin project](https://github.com/kykim/rem) says:

```
git clone https://github.com/kykim/rem.git
cd rem
xcodebuild
cp build/Release/rem /usr/local/bin
```

Also, I did test that.